runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/SidebarNavigationPanel/SidebarNavigationPanel.tsx

```diff
@@ -18,15 +18,16 @@ interface TabInfo {
 const SidebarNavigationPanel: FC<SidebarProps> = ({ onTabToggle: onTabChange, selectedTab }) => {
     const hasReadProcessInfoAccess = useFeatureKey([FeatureKey.PROCESS_INSTANCE_READ]);
     const { translate } = useTranslations();
+    const currentLanguage = i18n.language;
 
     const sidebarTabs = useMemo(() => {
         const tabs: TabInfo[] = [];
         if (hasReadProcessInfoAccess)
-        { tabs.push({ value: ProcessBuildTab.RUN_INFO, label: translate('Process.MainView.Sidebar.RunInfo') }); }
+        { tabs.push({ value: ProcessBuildTab.RUN_INFO, label: translate('Process.MainView.Sidebar.RunInfo') }, {value: ProcessBuildTab.PROCESS_VARIABLES, label: translate('Process.MainView.Sidebar.ProcessVariables')}); }
 
         return tabs;
         // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [hasReadProcessInfoAccess]);
+    }, [hasReadProcessInfoAccess, currentLanguage]);
```

Comment:
its not what I desired. you need to implement new const in `useTranslations.tsx` that will do it and use this hook here